grammars/css.cson

```diff
@@ -20,294 +20,1205 @@
+            ]
+          }
+          {
+            'begin': '{'
+            'beginCaptures':
+              '0':
+                'name': 'punctuation.section.supports.begin.css'
+            'end': '}'
+            'endCaptures':
+              '0':
+                'name': 'punctuation.section.supports.end.css'
+            'name': 'meta.at-rule.supports.body.css'
+            'patterns': [
+              {
+                'include': '$self'
+              }
+            ]
+          }
+        ]
+      }
+      {
+        # @viewport
+        'begin': '(?i)((@)viewport)(?=[\\s{\'";]|/\\*|$)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.viewport.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': '(?=[{;@])'
+        'name': 'meta.at-rule.viewport.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        # @font-feature-values
+        'begin': '(?i)((@)font-feature-values)(?=[\\s{\'";]|/\\*|$)\\s*'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.font-feature-values.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'contentName': 'variable.parameter.font-name.css'
+        'end': '\\s*(?={|[@;])'
+        'name': 'meta.at-rule.font-features.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        # @annotation, @character-variant, @ornaments, @styleset, @stylistic, @swash
+        'include': '#font-features'
+      }
+      {
+        # @namespace
+        'begin': '(?i)((@)namespace)(?=[\\s\'";]|/\\*|$)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.namespace.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': ';|(<=["\')])|(?=[@{])'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.terminator.rule.css'
+        'name': 'meta.at-rule.namespace.css'
+        'patterns': [
+          {
+            'include': '#url'
+          }
+          {
+            'captures':
+              '1':
+                'patterns': [
+                  {
+                    'include': '#comment-block'
+                  }
+                ]
+              '2':
+                'name': 'entity.name.function.namespace-prefix.css'
+                'patterns': [
+                  {
+                    'include': '#escapes'
+                  }
+                ]
+            'match': '''(?xi)
+              (?:\\G|^|(?<=\\s))
+              (?=
+                (?<=\\s|^)                             # Starts with whitespace
+                (?:[-a-zA-Z_]|[^\\x00-\\x7F])          # Then a valid identifier character
```

Comment:
Yeah, I changed it to use hex codes so it's compatible with PCRE/GitHub's syntax highlighting. Go figure.